src/collapse/docs/readme.md

```diff
@@ -7,3 +7,27 @@
   <i class="glyphicon glyphicon-eye-open"></i>
   _(Default: `false`)_ -
   Whether the element should be collapsed or not.
+
```

Comment:
Let's be honest in here. Some callbacks has default and other doesn't. Leave it as is, I prefer this way.